a10_octavia/controller/worker/tasks/handler_l7rule.py

```diff
@@ -83,53 +93,59 @@ def execute(self, l7rule, listeners, vthunder, update_dict):
 
 
 class DeleteL7Rule(BaseVThunderTask):
-    """ Task to delete a L7rule and disassociate from provided pool """
+
+    """Task to delete a L7rule and disassociate from provided pool """
 
     def execute(self, l7rule, listeners, vthunder):
-        """ Execute delete L7Rule """
+        """Execute delete L7Rule """
```

Comment:
Remove the space

Reply:
Removed this redundant line, as class is already defining exact same thing